src/mscorlib/src/System/Globalization/SortVersion.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization 
+{
+    using System;
+    using System.Diagnostics.Contracts;
+
+    [Serializable]
```

Comment:
Should it make sense for this and perhaps other parts of the new exposed APIs to go under the corefx code so that we can share with Unix and/or .NET Native?